R/output-selections.R

```diff
@@ -0,0 +1,38 @@
+#' @title OutputSelections

```

Comment:
Same hiearchy as in Core. OutputSelection is a prop of SimulationSettings which is defined in a simulation